### DISCUSSION TITLE
Denon AVR integration vs HEOS integration with Music Assistant

### DISCUSSION THREAD

**Discussion by prbtt** [music-assistant/support#494] [Q&A] [answered]
There are two integrations that allow you to control Denon AVR media players with home assitant. Those are denonavr or HEOS. Im curently using denonavr, since it allows me to create two separate media players for my multiroom setup. Unfortunately i wasnt able to use them together with Music Assistant. Did I do something wrong or it really isint supported? I guess it would work with HEOS but thats not ideal since it doesnt control zone 1 and zone 2 separately.

**Comment by marcelveldt** [comment, marked as answer]
The Denon integration in HA does not support playback at all.
Most people seem to be using the DLNA integration with their Denon receivers for playback of music (from Music Assistant)

**Reply by yoavbrog** [reply]
@prbtt were you able to solve this?
I too have a Denon receiver with 2 zones, and trying to use Music Assistant to use each zone as a separate player. 
Thanks!

**Comment by OzGav** [comment]
Main HEOS discussion here https://github.com/orgs/music-assistant/discussions/1167
